pyproject.toml

```diff
@@ -23,6 +23,7 @@ s3fs = "2025.3.2"
 pyjwt = "^2.10.0"
 msal-extensions = "^1.0.0"
 requests-aws4auth = {version="1.3.1", optional=true}
+miniwdl = "^1.13.0"
```

Comment:
This could be an optional dependency, under `extras`, and then lazy loaded if using the wdl specific code

Reply:
Done